packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -2062,6 +2113,21 @@ class AnnotatedRegionLayer<T> extends ContainerLayer {
   /// Ignored if [size] is not set.
   final Offset offset;
 
+  /// Whether this layer should prevent layers visually behind it from receiving
+  /// the hit testing.
+  ///
+  /// If [opaque] is true, it will "absorb" the hit test if any of its children
```

Comment:
"the hit test" -> "hits" (and "it" -> "them" on the next line)

Reply:
I changed it to "the hit". Describing the event during a single hit test might be easier to understand?